packages/webapp/src/_app/api/eden-contract.ts

```diff
@@ -1,7 +1,7 @@
 import { CONTRACT_GLOBAL_TABLE, getTableRows } from "_app";
 
 export const getIsCommunityActive = async (): Promise<boolean> => {
-    const rows = await getTableRows(CONTRACT_GLOBAL_TABLE, "community");
+    const rows = await getTableRows(CONTRACT_GLOBAL_TABLE, {lowerBound: "community"});
```

Comment:
Would you mind double checking your prettier auto-formatting? I believe there should be spaces inside of your curly braces here; autoformat would fix that.